3696/3/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'sinatra/flash'
+require 'truemail'
+
+class ApplicationController < Sinatra::Base
+  configure do
+    enable :sessions
+    set :session_secret, ENV.fetch('SECRET')
+  end
+
+  Truemail.configure do |config|
+    config.verifier_email = 'ojiknpe@net8mail.com'
+  end
+
+  helpers Sinatra::AuthHelper
+
+  use UserController
+  use PlaceController
+  use ReviewController
```

Comment:
Nice rack-like solution

Reply:
I'm not sure if it will work, but worth trying
```ruby
namespace('/reviews') { use ReviewController }
```

Reply:
![Screenshot from 2019-07-25 23-48-44](https://user-images.githubusercontent.com/41728508/61907799-e6653d00-af36-11e9-9b76-312f3f212526.png)
Nah

Reply:
http://sinatrarb.com/contrib/namespace

Reply:
Thanks, an interesting gem. Will use it in ReviewController file. Doesn't seem to work with use-statements, though:(

Reply:
Or works? :thinking: